test/setting-integration.test.js

```diff
@@ -17,6 +17,8 @@ import assert from 'assert';
 import { contextHelper } from '@adobe/content-lake-commons';
 import { SettingsStore } from '../src/settings.js';
 
+const TIMEOUT_5_SEC = 5000;
```

Comment:
nitpick: call it something generic `MOCHA_TEST_TIMEOUT` so you can easily increase/decrease it without having to change the name of the variable